contracts/src/v0.8/automation/dev/MercuryRegistry.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity 0.8.6;
+pragma solidity 0.8.19;
```

Comment:
Do these files need to be updated in this change?

Reply:
not necessary but automations team agreed to it . https://chainlink-core.slack.com/archives/C033D4HSSQJ/p1710877172545099